README.md

```diff
@@ -58,15 +58,15 @@ Here are a few other utility commands you may find useful:
 Sometimes you will want to use the latest version of the `web-design-standards` repo. Follow these steps to do so:
 
 1. Clone the latest version of the [`web-design-standards` repo](https://github.com/18F/web-design-standards/tree/staging).
-1. Run `npm install` to install the dependencies required for the package.
-1. Run `npm run build:package` to create the built version of the Standards.
+1. Run `npm install` to install the dependencies required for the package on the `web-design-standards` directory.
```

Comment:
I'd suggest "in the `web-design-standards` directory" to be consistent with the other steps